clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/converters/DeleteAmazonLoadBalancerAtomicOperationConverter.groovy

```diff
@@ -28,8 +31,22 @@ import org.springframework.stereotype.Component
 class DeleteAmazonLoadBalancerAtomicOperationConverter extends AbstractAtomicOperationsCredentialsSupport {
 
   @Override
-  DeleteAmazonLoadBalancerAtomicOperation convertOperation(Map input) {
-    new DeleteAmazonLoadBalancerAtomicOperation(convertDescription(input))
+  AtomicOperation convertOperation(Map input) {
+    DeleteAmazonLoadBalancerDescription description = convertDescription(input)
+    // Default to Classic ELBs
+    if (description.loadBalancerType == null) {
+      description.loadBalancerType = AmazonLoadBalancerType.ELB_CLASSIC
+    }
+
+    switch (description.loadBalancerType) {
+      case AmazonLoadBalancerType.ALB:
+        return new DeleteAmazonLoadBalancerV2AtomicOperation(description)
+        break
+      case AmazonLoadBalancerType.ELB_CLASSIC:
+      default:
```

Comment:
I'd probably just throw an exception on the default case otherwise if some new type of load balancer gets added it won't silently slip through here and end up handled as a classic ELB